test/filt_stream.jl

```diff
@@ -354,4 +408,7 @@ end
             test_arbitrary(Th, x, convert(Float64, ratio)+rand(), 32)
         end
     end
+    if numerator(ratio) == interpolation && denominator(ratio) == decimation && numerator(ratio) != 1 && denominator(ratio) == 1
```

Comment:
Could this just be `if decimation == 1` ?

Reply:
I believe you are right here. I couldn't figure out what the `numerator(ratio)==interpolation && denominator(ratio == decimation` was for in the test_rational if statement so I left them in case I was simply not understanding the purpose.